src/main/java/bio/terra/app/configuration/WebConfig.java

```diff
@@ -12,6 +18,9 @@
 
 @Component
 public class WebConfig implements WebMvcConfigurer {
+  private final Logger logger = LoggerFactory.getLogger(WebConfig.class);
+  private static final String DEFAULT_SWAGGER_UI = "5.17.14";
```

Comment:
I think it's possible to get this value from the `build.gradle` environment variable as a static Java variable. Something to think about so we don't have a discrepancy between this value and the one in the `build.gradle`, and we don't need to manually bump this when a new version is published.

Reply:
The only way I've been able to get values from `build.gradle` is via properties files, which is what this PR does. If you have a reference for your suggestion, I'd be happy to give it a try - my gradle experience is very old and out of date.

Reply:
In practice, when will this default be used? Instead of a fallback, could we have TDR fail to launch?

I think we should either require that the file be present or hardcode the version in our source code. Doing both seems like a worse solution as it adds extra code to store and read the version, but still requires keeping the version in the source up to date.

Reply:
If we have it error out the error should be clear as we already have a proliferation of misleading startup errors.